docs/en/04_advanced-configuration.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Advanced Configuration
```

Comment:
```suggestion
title: Advanced Static Publishing Configuration
```

Reply:
I don't agree with this - see https://github.com/silverstripe/silverstripe-realme/pull/192#discussion_r2218139490